### PR TITLE
Locator Tag: support for remaining targets

### PR DESCRIPTION
**Pull request by kapi-no**

### CODE REVIEW

**Review by mkapala-nordic — APPROVED** [review]

File: samples/bluetooth/fast_pair/locator_tag/Kconfig [hunk 1]
```diff
@@ -8,16 +8,16 @@ source "Kconfig.zephyr"
 
 menu "Nordic Fast Pair Locator tag sample"
 
-choice FAST_PAIR_LOCATOR_TAG_PLATFORM
-	prompt "Locator tag sample platform"
-	default FAST_PAIR_LOCATOR_TAG_PLATFORM_THINGY if BOARD_THINGY53_NRF5340_CPUAPP
-	default FAST_PAIR_LOCATOR_TAG_PLATFORM_DK
+choice APP_PLATFORM
+	prompt "Application platform"
```
Comment (MarekPieta) [inline samples/bluetooth/fast_pair/locator_tag/Kconfig:12]: Should it actually have prompts? There is no need to assign it by the user (property determined by the used board - a supported board could update default here I think).

File: samples/bluetooth/fast_pair/locator_tag/boards/thingy53_nrf5340_cpuapp.overlay [hunk 2]
```diff
@@ -4,15 +4,4 @@
...
-	};
-
-	pwmspk {
-		compatible = "pwm-leds";
-		pwm_spk0: pwm-spk-0 {
-			pwms = <&pwm1 0 PWM_MSEC(20) PWM_POLARITY_NORMAL>;
-		};
-	};
-};
+#include "thingy53_nrf5340_cpuapp_common.overlay"
```
Comment (MarekPieta) [inline samples/bluetooth/fast_pair/locator_tag/boards/thingy53_nrf5340_cpuapp.overlay:7]: It should be `*.dtsi` file (DTS include). That would separate naming from board-specific overlays and clearly show the intention here.

File: samples/bluetooth/fast_pair/locator_tag/Kconfig (hunk 1 above)
Comment (MarekPieta) [reply samples/bluetooth/fast_pair/locator_tag/Kconfig:12]: Actually maybe we can even rely directly on board Kconfig here?

**Review by MarekPieta — CHANGES_REQUESTED** [review]

**Comment by kapi-no** [comment]
Adding @divipillai due to one minor improvement in the sample Readme (got the "doc-required" label)

**Review by divipillai — APPROVED** [review]

**Review by MarekPieta — APPROVED** [review]

**Comment by kapi-no** [comment]
Rebased my PR to pass CI. I haven't changed this PR file diff

**Pull request merged**